wrap.js

```diff
@@ -0,0 +1,55 @@
+// Wrap
+// wraps a string by a certain width
+
+makeString = require('./helper/makeString');
+
+module.exports = function wrap(str, width, seperator, cut){
+	str = makeString(str);
+	width = width || 75;
+	seperator = seperator || '\n';
+	cut = cut || false;
+	if(width <= 0){
+		return str;
+	}
+	else if(!cut){
+		words = str.split(" ");
+		for(var i = 0; i < words.length - 1; i++){
+			//preserve the spaces for all words except the last one
```

Comment:
I'm actually not sure if we should preserve the space. This behaviour is somehow different than I would expect it to be.